src/backend/main.ts

```diff
@@ -266,6 +220,13 @@ async function createWindow(): Promise<BrowserWindow> {
       autoUpdater.checkForUpdates()
     }
   }
+
+  ipcMain.on('setZoomFactor', async (event, zoomFactor) => {
+    mainWindow.webContents.setZoomFactor(
```

Comment:
moved this inside since it depends on the mainWindow object that was just created